packages/vite/src/runFeServer.ts

```diff
@@ -119,10 +119,25 @@ export async function runFeServer() {
     app.get(expressPathDef, routeHandler)
   }
 
-  app.listen(rwConfig.web.port)
-  console.log(
-    `Started production FE server on http://localhost:${rwConfig.web.port}`
+  const server = app.listen(
+    rwConfig.web.port,
+    process.env.NODE_ENV === 'production' ? '0.0.0.0' : '::'
```

Comment:
I think no need to do this toggle maybe, since runFeServer is always production, no?

Reply:
No, I had to specifically do `NODE_ENV=production <command>` when I ran this locally to test

Reply:
Well, so yeah, maybe if we always do `0.0.0.0` we don't need to toggle, if that's what you meant

Reply:
Yes that's what I mean :). Because runFeServer is only used for production serving, never in development!

Reply:
I guess that could be said for all our serve commands in that case. We can think more about how we want to handle this when all this gets rewritten for the ongoing server-file stuff